e107_handlers/form_handler.php

```diff
@@ -1024,11 +1024,11 @@ function avatarpicker($name,$curVal='',$options=array())
 		if($count == 0)
 		{
 			$text .= "<div class='row'>";
-			$text .= "<div class='alert alert-info'>No Avatars Available</div>"; //TODO LAN
+			$text .= "<div class='alert alert-info'>".LAN_EFORM_005."</div>"; 
 
 			if(ADMIN)
 			{
-				$text .= "<div class='alert alert-danger'>Admin-Only Notice: The folder <b>".e_AVATAR_DEFAULT."</b> is empty. Upload some default avatars images to this folder for users to choose avatars from.</div>"; //TODO LAN
+				$text .= "<div class='alert alert-danger'>".LAN_EFORM_006." <b>".e_AVATAR_DEFAULT."</b> ".LAN_EFORM_007."</div>"; 

```

Comment:
Please use lanVars. (ie. never split a sentence with LANs) 
"Admin-Only Notice: The folder [x] is empty. Upload some default avatars images to this folder for users to choose avatars from."